ext/hal/st/stm32cube/stm32l0xx/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2017 Erwin Rol <erwin@erwinrol.com>
+# Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
```

Comment:
This is not the correct location of the change, please do it in the previous patch